templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -68,7 +68,10 @@
             :active="tab === 'appearance'"
             @click="changeTab('appearance')"
           >
-            <appearance-form :node="node" />
+            <appearance-form
+              :node="node"
+              :is-page-child="isPageMultiConentNodeChild"
```

Comment:
Isn't the prop named `isPageChild` ? not kebab-case?

Reply:
I believe vue/js knows how to do that transformation by itself because it is common to use kebab-case for tags but JS properties are written camelCase